src/client/datascience/kernel-launcher/localKernelFinder.ts

```diff
@@ -109,12 +108,24 @@ export class LocalKernelFinder implements ILocalKernelFinder {
                 preferredInterpreter,
                 undefined
             );
+            sendTelemetryEvent(Telemetry.PreferredKernel, undefined, {
+                result: preferred ? 'found' : 'notfound',
```

Comment:
This will give us a better idea of how often we find the right match.
Its difficult to tell from the logs right now whether we failed or didn't find, or other